File: BackEnd/CSharp/ClientSide.cs
namespace WebApp
{
    class ClientSide
    {
        public string DeleteFile(string Paramaters)
        {
            // MsgBox("DeleteFile-Paramaters:" + Paramaters)
            return "alert('Deleted File: " + Paramaters + "');";
        }

        public string SomethingElse(string Paramaters)
        {
            // MsgBox("SomethingElse-Paramaters:" + Paramaters)
            return "SomethingElse";
        }
    }
}

File: BackEnd/VB.NET/ClientSide.vb
Public Class ClientSide
    Function DeleteFile(Paramaters As String) As String
        'MsgBox("DeleteFile-Paramaters:" + Paramaters)
        Return "alert('Deleted File: " + Paramaters + "');"
    End Function
    Function SomethingElse(Paramaters As String) As String
        'MsgBox("SomethingElse-Paramaters:" + Paramaters)
        Return "SomethingElse"
    End Function
End Class
